meinberlin/apps/projects/tasks.py

```diff
@@ -191,5 +192,7 @@ def reset_cache_for_projects(starts: bool, ends: bool) -> str:
             msg += "failed for active projects becoming past"
         else:
             msg += "succeeded for active projects becoming past"
+    # update virtual table view
+    public_projects_exec_raw_query()
```

Comment:
when clearing the cash, the raw query it reconstructs the virtual table. But we could also make a separate task to do that few times in a day.